### DISCUSSION TITLE
v2.15.3

### DISCUSSION THREAD

**Discussion by Mygod** [Mygod/VPNHotspot#443] [General]
* Brazilian Portuguese support by <eduardofortunato569@gmail.com>.
* Non-Google Play version can now use repeater/temporary hotspot without location enabled as well.
* Fixes.

**Full Changelog**: https://github.com/Mygod/VPNHotspot/compare/v2.15.2...v2.15.3

<hr /><em>This discussion was created from the release <a href='https://github.com/Mygod/VPNHotspot/releases/tag/v2.15.3'>v2.15.3</a>.</em>

**Comment by VeraxLee** [comment]
crashed on xiaomi k40s (poco f4)

system version is miui 13.2.8 (android 13)

**Reply by Mygod** [reply]
Logcat?

**Reply by VeraxLee** [reply]
sorry but...how to do it?

**Reply by Mygod** [reply]
You could send the crash report to Google Play when the app crashes, or you could get a logcat app and get logcat after crash from there.

**Reply by VeraxLee** [reply]
thank u.
I send crash report to google play just before.

**Reply by VeraxLee** [reply]
![Screenshot_2022-12-24-13-47-14-742_be.mygod.vpnhotspot.jpg](https://user-images.githubusercontent.com/80865701/209423300-511255e3-6b09-42b0-b02f-0f8c94821419.jpg)

**Reply by Mygod** [reply]
Unfortunately I do not see this issue on my end. Please try grabbing logcat manually.

**Reply by Mygod** [reply]
Okay I can see it now. It seems like a Google Play issue. Please do not use app cloning or anything like that.

**Reply by VeraxLee** [reply]
> Okay I can see it now. It seems like a Google Play issue. Please do not use app cloning or anything like that.

Hello sir Now I update my phone to MIUI 14.0.3 and everything is OK now！！！ I am so excited！！！

I'm wondering what makes the difference. Both miui14 and miui13.2.8 are android 13...  So strange.

**Reply by VeraxLee** [reply]
> Okay I can see it now. It seems like a Google Play issue. Please do not use app cloning or anything like that.

Oh,  Now I checked that this problem is because of xposed app“hide my applist” version 3.06,from https://github.com/Dr-TSNG/Hide-My-Applist
